tuning/autotune.py

```diff
@@ -259,11 +281,35 @@ def calculate_md5(file_path: str) -> str:
     return md5.hexdigest()
 
 
+def find_collisions(lst):
```

Comment:
1. Add type hints
2. The argument name is not very descriptive, could we use a more verbose name instead